app/models/solr_document.rb

```diff
@@ -26,41 +24,11 @@ class SolrDocument
   # Recommendation: Use field names from Dublin Core
   use_extension( Blacklight::Document::DublinCore)
 
-  # Aggregate item in context urls for this solr doc and its children, returning a hash that maps pids to item in context urls
-  def self_and_child_pids_to_item_in_context_urls
-    return @self_and_child_pids_to_item_in_context_urls ||= begin
-      pids_to_urls = {}
-      pids_to_urls[self['id']] = item_in_context_url if item_in_context_url.present?
-
-      pids_to_urls.merge(child_pids_to_item_in_context_urls)
-    end
-  end
-
   # Item in context url for this solr document. Might return nil if this doc has no item in context url.
   def item_in_context_url
     self['lib_item_in_context_url_ssm'].present? ? self['lib_item_in_context_url_ssm'].first : nil
   end
 
-  def child_pids_to_item_in_context_urls
```

Comment:
These methods were unnecessary given the way that child documents are now fetched in show views.